translations/ui-data-export/en.json

```diff
@@ -10,7 +14,9 @@
   "viewAllLogs": "View all",
   "uploaderTitle": "Drag and drop",
   "mappingProfilesTitle": "Field mapping profiles",
+  "mappingProfilesTitle.manager": "Data export settings - Field mapping profiles",
   "jobProfilesTitle": "Job profiles",
+  "jobProfilesTitle.manager": "Data export settings - Job profiles",
```

Comment:
Use case consistently. The new lines 5 and 6 use `- lower` but 17 and 19 use `- Upper`. I don't think one is more "correct"; just pick one.

Reply:
thanks for notice @zburke, I have missed it in developments phase)